bbb-graphql-server/metadata/databases/BigBlueButton/tables/public_v_user_ref.yaml

```diff
@@ -6,6 +6,16 @@ configuration:
   custom_column_names: {}
   custom_name: user_ref
   custom_root_fields: {}
+object_relationships:
+  - name: reaction
+    using:
+      manual_configuration:
+        column_mapping:
+          userId: userId
+        insertion_order: null
+        remote_table:
+          name: v_user_reaction_current
```

Comment:
Hold on, I'm changing how we query the user reaction emoji in order to optimize it for better performance!
I'm sending a PR soon and then you can make use of the new column!

Reply:
https://github.com/bigbluebutton/bigbluebutton/pull/20449

Reply:
Nice!

Reply:
Once it is no longer necessary don't forget to revert the changes in this file!